docs/postprocessing/index.rst

```diff
@@ -0,0 +1,15 @@
+Postprocessing
+==============
+
+After the simulation is complete, either by running the **run_all.bat** script or following the step-by-step procedure, for postprocessing, the following steps are to be followed:
+
+1. Install `R <https://www.r-project.org/>`_ based on the OS
+
+2. Get the path to the R executable, e.g. **c:\\"Program Files"\R\R-4.2.1\bin\R.exe**, on Windows
```

Comment:
is there a way to link to an external article that teaches them how to get the path

Reply:
Yes, I will include links based on the OS